app/controllers/api/vcs.php

```diff
@@ -544,8 +557,9 @@
         ]), Response::MODEL_VCS_CONTENT_LIST);
     });
 
-App::post('/v1/vcs/github/installations/:installationId/providerRepositories/:providerRepositoryId/detection')
-    ->desc('Detect runtime settings from source code')
+App::post('/v1/vcs/github/installations/:installationId/providerRepositories/:providerRepositoryId/detections')
```

Comment:
`providerRepositories` -> `provider-repositories`

Reply:
Alternative is to have just a simple

```
/v1/vcs/github/installations/:installationId/detections
```

And accept providerRepositoryId as parameter. Let's suggest it, see which approach is more liked by Eldad/Christy

Reply:
Christy approved this one: `/v1/vcs/github/installations/:installationId/detections`